modules/apps/commerce/commerce-frontend-js/test/test_utilities/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export function getRandomInt(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
```

Comment:
I don't think you need these `ceil` and `floor` calls, as you control all the call sites and you're only ever passing integers.